solo/admin.py

```diff
@@ -36,42 +40,44 @@ def get_urls(self):
             model_name = self.model._meta.module_name.lower()
 
         self.model._meta.verbose_name_plural = self.model._meta.verbose_name
-        url_name_prefix = '%(app_name)s_%(model_name)s' % {
-            'app_name': self.model._meta.app_label,
-            'model_name': model_name,
+        url_name_prefix = "%(app_name)s_%(model_name)s" % {
+            "app_name": self.model._meta.app_label,
+            "model_name": model_name,
         }
         custom_urls = [
-            re_path(r'^history/$',
-                    self.admin_site.admin_view(self.history_view),
-                    {'object_id': str(self.singleton_instance_id)},
-                    name='%s_history' % url_name_prefix),
-            re_path(r'^$',
-                    self.admin_site.admin_view(self.change_view),
-                    {'object_id': str(self.singleton_instance_id)},
-                    name='%s_change' % url_name_prefix),
+            re_path(
+                r"^history/$",
```

Comment:
Please revert the changes in string quotes so that this diff is much smaller and easier to maintain.

Reply:
Argh... I use black as default formatter, it uses double quotes as default quoting scheme - I'll update it tomorrow morning !

Reply:
`black` is great and I use it my projects as well. But since `django-solo` doesn't yet, let's keep the diff small. Thanks.